nodejs/tfwconnector/TWFConnector.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+const zmq = require('zeromq'),
+    subSock = zmq.socket('sub'),
+    pushSock = zmq.socket('push');
+
+const subPort = process.env.TFW_PUBLISHER_PORT || 7654,
+    pushPort = process.env.TFW_RECEIVER_PORT || 8765;
+
+const subAddress = "tcp://localhost:" + subPort,
+      pushAddress = "tcp://localhost:" + pushPort;
+
+
+class TFWConnector {
+    constructor() {
+        this.pushSocket = pushSock.connect(pushAddress);
+        this.subSocket = subSock.connect(subAddress);
+        console.log("Push socket connected to: " + pushAddress + "\n" + "Subscribe socket connected to: " + subAddress);
+    }
+
+    subscribe(key) {
+        this.subSocket.subscribe(key);
+    }
+
+    sendMessage(message, name = "avataobot") {
```

Comment:
This should be a separate MessageSender class using TFWConncetor to maintain API consistency (check the way it is done in the `python3` subdirectory)